velox/exec/tests/TableScanTest.cpp

```diff
@@ -3696,6 +3696,29 @@ TEST_F(TableScanTest, aggregationPushdown) {
   EXPECT_EQ(0, loadedToValueHook(task));
 }
 
+TEST_F(TableScanTest, decimalDisableAggregationPushdown) {
+  auto rowType = ROW({"c0", "c1"}, {BIGINT(), DECIMAL(18, 2)});
+  auto vectors = makeVectors(10, 1'000, rowType);
+  auto filePath = TempFilePath::create();
+  writeToFile(filePath->getPath(), vectors);
+  createDuckDbTable(vectors);
+  auto op = PlanBuilder()
+                .tableScan(rowType)
+                .singleAggregation({"c0"}, {"min(c1)", "max(c1)", "sum(c1)"})
+                .planNode();
+  auto task = assertQuery(
```

Comment:
@Yuhta This assert can't pass. Should I exclude decimal type from duckdb? Could you give me some guidance? Thanks!